operations/boot_orchestration/Cheatsheet.md

```diff
@@ -34,7 +34,7 @@ cray bos v2 sessions create --template-name SESSION_TEMPLATE_NAME --operation Re
 
 ### Single node commands (v2)
 
-(`ncn-mw#`) Boot a single node:
+c
```

Comment:
is this intentional?

Reply:
No; apparently I missed a file in the recent merge. Fixing.